### ISSUE TITLE
how to load dataset

### ISSUE THREAD

**Issue by lylyjy** [alessiabertugli/AC-VRNN#1]
I don't know how to run heatmap.py.
How to define data_dir?
Thanks.

**Comment by alessiabertugli** [comment]
Hi,

You have to download the dataset that you want to use and set data_dir to your local directory containing the dataset.

For ETH/UCY there are 3 versions as explained in the paper.
A) The dataset version used by SGAN and STGAT. You can download it following the original project: https://github.com/agrimgupta92/sgan/blob/master/scripts/download_data.sh.
B) The dataset version used by SR-LSTM, in which the authors modified Biwi ETH annotations due to a frame error: https://github.com/zhangpur/SR-LSTM.
C) The dataset version used by Social-Ways, in which the authors do not use the leave-one-out protocol but simply split each scene as 4/5 for the training set and 1/5 for the test set. You can prepare the dataset following the original project: https://github.com/crowdbotp/socialways.

Finally, if you want to use Stanford Drone Dataset, you can download the World Plane Human-Human Dataset from http://trajnet.stanford.edu/data.php?n=1, download SDD, and use dataset_processing/split_sdd.py to process the data for this code.

Best regards,
Alessia
